Makefile

```diff
@@ -1,19 +1,51 @@
 SHORT_NAME := object-storage-cli
 REPO_PATH := github.com/arschles/${SHORT_NAME}
```

Comment:
few places in the repo still uses it https://github.com/deis/object-storage-cli/blob/master/actions/download.go#L8.
Created a new issue #5 to track this.